src/assets/css/index.scss

```diff
@@ -205,8 +213,8 @@ $primary-color: black;
   list-style-type: none;
   text-align: left;
   padding: 1%;
```

Comment:
Remove this line, because the style attributes below override it, or remove the lines below and refactor them to one line with the style `padding: top right bottom left`

Reply:
Cool, thanks,,,I missed that